pkg/middleware/privilegetoken/claims.go

```diff
@@ -1,63 +1,22 @@
 package privilegetoken
 
 import (
-	"encoding/json"
-	"fmt"
+	"math/big"
 
 	"github.com/DIMO-Network/shared/pkg/privileges"
 	"github.com/ethereum/go-ethereum/common"
-	"github.com/ethereum/go-ethereum/common/hexutil"
-	"github.com/gofiber/fiber/v2"
 	"github.com/golang-jwt/jwt/v5"
-	"google.golang.org/protobuf/types/known/structpb"
 )
 
+// TODO: move this to token-exchanges
 type CustomClaims struct {
 	ContractAddress common.Address         `json:"contract_address"`
-	TokenID         string                 `json:"token_id"`
+	TokenID         *big.Int               `json:"token_id"`
```

Comment:
note: On the face of it, this is not backwards-compatible and probably dangerous in JS environments.

On the other hand we don't have any JS code that reads this right now and the tokens don't last very long. But we'd want to roll out upgrades to all affected services fairly quickly. (Or support both formats somehow).